src/components/form-helpers/form-helpers.js

```diff
@@ -348,6 +349,28 @@ export const InputFieldItem = ({
   </FieldItem>
 );
 
+export const NameFieldItem = ({
+  className,
+  fieldName,
+  label,
+  type,
+  placeholder,
+  required,
+  ...rest
+}) => (
+  <FieldItem className={concatClasses('field-item', className)} fieldName={fieldName}>
+    <label htmlFor={fieldName}>{label}</label>
+    <Field
+      type="text"
+      name={fieldName}
+      id={fieldName}
+      placeholder={placeholder}
+      validate={combineValidations(createRequiredValidation(required), validateName)}
```

Comment:
What do you think about adding another flag to _InputFieldItem_ in order to reduce duplicated code?

Reply:
Yes i already push this changes